### PR TITLE
[Refactor] Add support for validating multiple manifests simultaneously

### PR DESCRIPTION
**Pull request by humblenginr**
**Description**

This PR adds support for multiple manifests to be validated at the same time. This avoids having to send multiple requests to get things validated. 

**[Signed commits](https://github.com/meshery/meshery/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Review by leecalcote — CHANGES_REQUESTED** [review]
Do the REST API docs need an update?

File: handlers/oam_handler.go [hunk 1]
```diff
@@ -68,6 +68,32 @@ func getAppropriateCueVal(value string, valType string) (cue.Value, error) {
...
+	schema, err := getAppropriateCueVal(val.Schema, val.SchemaType)
+	if err != nil {
+		return false, err
+	}
+	value, err := getAppropriateCueVal(val.Value, val.ValueType)
+	if err != nil {
+		return false, err
+	}
+	isValid, err := utils.Validate(schema, value)
+	return isValid, err
```
Comment (Revolyssup) [inline handlers/oam_handler.go:81]: just return utils.Validate(schema, value) will reduce one extra line.

File: handlers/oam_handler.go [hunk 2]
```diff
@@ -76,7 +102,7 @@ func getAppropriateCueVal(value string, valType string) (cue.Value, error) {
 // 	200:
 
 // request body should be json
-// request body should be of format - {schema: string, schemaType: "JSON" | "YAML" | "JSONSCHEMA", value: string, valueType: "JSON" | "YAML" }
+// request body should be of format - {schema: string, schemaType: "JSON" | "YAML" | "JSONSCHEMA", value: string, valueType: "JSON" | "YAML" } or {list: []validationInfo}
 // it will respond with error if any occurred, or with `isValid: bool` json
 func (h *Handler) ValidationHandler(rw http.ResponseWriter, r *http.Request) {
```
Comment (Revolyssup) [inline handlers/oam_handler.go:107]: This handler is part of coming `MeshModel` and not used currently anywhere right?

Comment (sudo-NithishKarthik) [reply handlers/oam_handler.go:107]: For the most part, yes. It is also being used in MeshMap [here](https://github.com/layer5labs/meshery-extensions/pull/1006/files#diff-6c003e5b640a294ef99c2c9e2509e29c38d9671784fe39373905bfd24c6adc08R8)

File: handlers/oam_handler.go (hunk 1 above)
Comment (sudo-NithishKarthik) [reply handlers/oam_handler.go:81]: 👍

**Comment by leecalcote** [comment]
> > Do the REST API docs need an update?
> 
> Yes.

Holding for API docs update.

**Comment by leecalcote** [comment]
> > > Do the REST API docs need an update?
> > 
> > 
> > Yes.
> 
> Holding for API docs update.

Waiting on API docs update.

**Pull request merged**